# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eglibc/x86-i586.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

2007-09-13 H.J. Lu <hongjiu.lu@intel.com>

* sysdeps/i386/i586/memcpy_chk.S: New.
* sysdeps/i386/i586/mempcpy_chk.S: Likewise.
* sysdeps/i386/i586/memset_chk.S: Likewise.

--- ./libc/sysdeps/i386/i586/memcpy_chk.S.i586 2007-10-02 07:00:06.000000000 -0700
+++ ./libc/sysdeps/i386/i586/memcpy_chk.S 2007-10-02 06:58:51.000000000 -0700
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memcpy_chk.S>
--- ./libc/sysdeps/i386/i586/mempcpy_chk.S.i586 2007-10-02 07:00:06.000000000 -0700
+++ ./libc/sysdeps/i386/i586/mempcpy_chk.S 2007-10-02 06:58:51.000000000 -0700
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/mempcpy_chk.S>
--- ./libc/sysdeps/i386/i586/memset_chk.S.i586 2007-10-02 07:00:06.000000000 -0700
+++ ./libc/sysdeps/i386/i586/memset_chk.S 2007-10-02 06:58:51.000000000 -0700
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memset_chk.S>
